Add switchbot curtain movement speed docs (#41503)
home-assistant/home-assistant.github.io: source/_integrations/switchbot.markdown

- get battery level
- get calibration state

Options:

Curtain devices have a slow mode option available in the configuration. However, slow mode is primarily designed for Curtain 3 models. Older curtain models (version 1 & 2) may ignore the slow mode setting.

1. To enable slow mode, go to {% my integrations title="**Settings** > **Devices & services**" %}.
2. Under **Integration entries**, find the curtain device and select **Configure**.
3. In the **Options** dialog, enable **Curtain slow mode**.

> 

Reviewer: I think this has to be moved over to the “Curtain 3” and/or “Curtain” sections, and the feature added under the “Features” lists.

I don’t know if the detailed steps are actually required, if that’s the case and it concerns both “Curtain 3” and “Curtain”, then maybe they can be moved under the “Cover” section instead? Otherwise I think a `- enable slow mode (may be ignored depending on the model of your curtain)` in the “Curtain” feature list would be fine.

- get light level
- get battery level
- get calibration state

> - enable slow mode

Reviewer 2: Please update the text after the core PR changes.

> - enable slow mode

Reviewer 3: ```suggestion
- enable curtain movement speed
```

> - enable curtain movement speed

Reviewer 2: ```suggestion
- set curtain movement speed
```

> Curtain slow mode is configured from the device options. Slow mode is primarily designed for Curtain 3 models, and older Curtain models may ignore the setting.

Reviewer 3: ```suggestion
Curtain movement speed is configured from the device options. Curtain movement speed is primarily designed for Curtain 3 models. Older Curtain models may ignore the setting.
```

> 1. To enable slow mode, go to {% my integrations title="**Settings** > **Devices & services**" %}.

Reviewer 3: ```suggestion
1. To enable **Curtain movement speed**, go to {% my integrations title="**Settings** > **Devices & services**" %}.
```

> 1. To enable **Curtain movement speed**, go to {% my integrations title="**Settings** > **Devices & services**" %}.

Reviewer 2: ```suggestion
1. To set **Curtain movement speed**, go to {% my integrations title="**Settings** > **Devices & services**" %}.
```

2. Find the SwitchBot integration and select the curtain device you want to configure.
3. Select **Configure** for that device.

> 4. In the **Options** dialog, enable **Curtain slow mode**.

Reviewer 3: ```suggestion
4. In the **Options** dialog, enable **Curtain movement speed**.
```

> 4. In the **Options** dialog, enable **Curtain movement speed**.

Reviewer 2: ```suggestion
4. In the **Options** dialog, set **Curtain movement speed** to a number between 0-255. The default is 255.
```
